src/generic/modal-dropzone/ModalDropzone.test.jsx

```diff
@@ -112,20 +116,20 @@ describe('<ModalDropzone />', () => {
 
     expect(response.asset.url).toBe(mockUrl);
 
-    const { getByRole, getByAltText } = render(<RootWrapper {...props} />);
+    const { getByRole, getByLabelText } = render(<RootWrapper {...props} />);
     const dropzoneInput = getByRole('presentation', { hidden: true }).firstChild;
     const uploadButton = getByRole('button', { name: messages.uploadModal.defaultMessage });
 
-    userEvent.upload(dropzoneInput, file);
+    await user.upload(dropzoneInput, file);
 
     await waitFor(() => {
       expect(uploadButton).not.toBeDisabled();
     });
 
-    userEvent.click(uploadButton);
+    await user.click(uploadButton);
 
-    await waitFor(() => {
-      expect(getByAltText(messages.uploadImageDropzoneAlt.defaultMessage)).toBeInTheDocument();
+    await act(() => {
+      expect(getByLabelText(messages.uploadImageDropzoneAlt.defaultMessage)).toBeInTheDocument();
     });
```

Comment:
```suggestion
    expect(getByLabelText(messages.uploadImageDropzoneAlt.defaultMessage)).toBeInTheDocument();
```
`act` is not a promise and this isn't making any action. When I tried with neither `act` nor `waitFor`, it was passing, but if something is necessary it would be `waitFor` not `act`.